### ISSUE TITLE
links don't work with Jaws screenreader

### ISSUE THREAD

**Issue by Veyfeyken** [adobe-accessibility/Accessible-Mega-Menu#21]
The links don't work with Jaws in IE. This makes the menu inaccessible with the most commonly used screenreader - browser combination worldwide.

Fails with: 
- Jaws 14 in IE9 on Windows 7
- Jaws 15 in IE11 on Windows 7

The problem doesn't occur with Jaws in Firefox.
Also works nicely with VoiceOver and NVDA.
